libs/sync/atomic_bool_test.go

```diff
@@ -0,0 +1,36 @@
+package sync
+
+import "testing"
+
+func TestDefaultValue(t *testing.T) {
```

Comment:
I might also add a few tests that attempt to modify the structure a bunch of times from multiple threads, in a way that the race detector would catch. The interesting part of this type is that it's thread safe/atomic, not that it's a boolean, and it would be nice if the tests would help provide confidence there.